RIOT-OS/RIOT
# driver/at86rf215: cleanup

### Contribution description

- prefix all compile-time configuration values with `CONFIG_`
- allow to disable modulations with `DISABLE_MODULE` 

### Testing procedure

This is just a compile-time cleanup, no run-time changes.

You should notice a reduction in code size when you add, e.g.

    DISABLE_MODULE += netdev_ieee802154_mr_oqpsk

to your Makefile

### Issues/PRs references
